patterns/services-three-column-boxed-pricing-table.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Title: Services - Three-column boxed pricing table
+ * Slug: twentytwentyfive/services-three-column-boxed-pricing-table
+ * Categories: call-to-action, banner, featured, services, pricing
+ * Description: A three-column boxed pricing table designed to showcase services, descriptions, and pricing options.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50)"><!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"60%","layout":{"type":"constrained","justifyContent":"left"}} -->
+<div class="wp-block-column" style="flex-basis:60%"><!-- wp:heading -->
+<h2 class="wp-block-heading">Choose your membership</h2>
+<!-- /wp:heading --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"center","width":"40%","layout":{"type":"constrained","justifyContent":"right"}} -->
+<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:40%"><!-- wp:buttons {"align":"wide","layout":{"type":"flex","justifyContent":"right","orientation":"horizontal","flexWrap":"nowrap"}} -->
+<div class="wp-block-buttons alignwide"><!-- wp:button {"className":"is-style-outline","style":{"border":{"style":"solid","width":"1px"},"typography":{"fontStyle":"normal","fontWeight":"400"}},"fontSize":"small","borderColor":"contrast"} -->
+<div class="wp-block-button has-custom-font-size is-style-outline has-small-font-size" style="font-style:normal;font-weight:400"><a class="wp-block-button__link has-border-color has-contrast-border-color wp-element-button" style="border-style:solid;border-width:1px" rel="#">Become a member</a></div>
+<!-- /wp:button --></div>
```

Comment:
I think the use of button here is smart, but I think before the theme is finalized, this will be a block style variation on the paragraph block, it is still being discussed in other pull requests how to solve this. So it might take some additional time before this PR can be merged.

Reply:
Thanks for the feedback; I'll follow the discussion and update the implementation once a decision is made.

Reply:
Hi, the pill style variation for the paragraph block has been merged.
It needs to be placed inside a row, or it will be too wide, so I suggest trying to replace the columns with a row.
Please test it, you may need to make further changes than only copy and pasting this. 😃 

```
<!-- wp:group {"align":"wide","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
<div class="wp-block-group alignwide"><!-- wp:heading -->
<h2 class="wp-block-heading">Choose your membership</h2>
<!-- /wp:heading -->

<!-- wp:paragraph {"className":"is-style-pill"} -->
<p class="is-style-pill">Become a member</p>
<!-- /wp:paragraph --></div>
<!-- /wp:group -->
```

Reply:
Thanks for your guidance. I have resolved the pill style issues. Please review and let me know if any changes are needed